### PR TITLE
[ALLI-5682] Record image changes

### PR DESCRIPTION
**Pull request by LuomaJuha**
Opened pull request for keeping track of needed things to check
TODO:
/* Record page changes */
- [x] Record page no image working
- [x] Record page image working
- [x] Paginator working
- [x] Record page style set
- [x] Record page translations
- [x] Show fewer items on certain widths
- [x] Remove lines

- [x] Remove current first and last

/* Modal */
- [x] Modal non zoomable styles set 
- [x] Modal zoomable styles set
- [x] Jump to next paginator if there is no image set
- [x] Image information area working

- [x] Image no longer flies to area, now goes smoothly

- [x] TÄMÄN AINEISTON TARJOAA with smaller font
- [x] Add spaces to image index info slash
- [x] Translations
- [x] Bigger images
- [x] Gallery buttons to another picture

/* Lists General */
- [x] Stylings

- [x] Buttons look bad against gray background

- [x] Align paginated image numbers

/* List */
- [x] List paginator working

/* List grid */
- [x] Masonry now correctly works, fixes a problem with old version where image was bigger
- [x] Paginator works

/* List condensed */
- [x] paginator works

/* Favorites list * /

/* General */
- [x] Move excess html from js
- [x] Settings to a seperate member so that extend can be used
- [x] Set default value of initialized references to null instead of ''
- [x] Set better values to loop, checking for images to load
- [x] Translations
- [x] Check all image sizes
- [x] revamp masonry and videojs checks

### CODE REVIEW

File: themes/finna2/js/finna-image-paginator.js [hunk 1]
```diff
@@ -0,0 +1,791 @@
...
+  ' </div>' +
+  ' <div class="record-informations">' +
+  '   <div class="record-index">' +
+  '     <span class="total"></span>' +
+  '   </div>' +
+  '   </div>' +
+  ' </div>' +
+  '</div>';
+
+  var previousRecordButton = '<button class="mfp-arrow mfp-arrow-left previous-record" type="button"><</button>';
```
Comment (EreMaijala) [inline themes/finna2/js/finna-image-paginator.js]: < pitäisi olla &lt;

File: themes/finna2/js/finna-image-paginator.js [hunk 2]
```diff
@@ -0,0 +1,791 @@
...
+  var nextRecordButton = '<button class="mfp-arrow mfp-arrow-right next-record" type="button">></button>';
```
Comment (EreMaijala) [inline themes/finna2/js/finna-image-paginator.js]: &gt;

File: themes/finna2/js/finna-image-paginator.js [hunk 3]
```diff
@@ -0,0 +1,791 @@
...
+    _.paginatorIndex = paginatorIndex;
+    _.isList = isList;
+    _.images = images;
+    _.root = $(paginatedArea);
+    _.root.removeClass('paginate');
+    
+    _.trigger = _.root.find('.image-popup-trigger');
+    _.trigger.attr('paginator-index', paginatorIndex++);
+    // Lets get all the data from the settings
+    _.recordId = settings.recordId;
```
Comment (EreMaijala) [inline themes/finna2/js/finna-image-paginator.js]: Jos asetukset laittaisi erilliseen memberiin, niin voisi käyttää jQueryn extend():iä niiden laittamiseen (ks. vaikka finna-map.js:22)

File: themes/finna2/templates/RecordDriver/DefaultRecord/record-image-popup-information.phtml [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+?>
+<?=
+$this->piwik([
+  'lightbox' => true, 'record' => $this->driver, 'action' => 'imagePopup',
+  'recordUrl' => $this->recordLink()->getUrl($this->driver)
+]);
+?>
+<script>
+    finna.layout.initOrganisationPageLinks();
+</script>
```
Comment (EreMaijala) [inline themes/finna2/templates/RecordDriver/DefaultRecord/record-image-popup-information.phtml]: rivinvaihto uupuu

**Review by EreMaijala — CHANGES_REQUESTED** [review]

**Review by samuli — APPROVED** [review]

**Pull request merged**